packages/html/src/listview/listview.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import { classNames } from '../utils';
+
+export interface ListViewProps {
+    className?: string;
+    children?: any;
+    itemsCount?: number;
+    hasItemsNumbers?: boolean;
+    focusItem?: Array<number>;
+    selectedItem?: Array<number>;
+    header?: boolean;
+    footer?: boolean;
+    pageable?: boolean;
+    pagerPosition?: 'top' | 'bottom';
+    bordered?: boolean;
+    borders?: 'all' | 'horizontal' | 'vertical';
+    layout?: 'flex' | 'grid';
+    flexDirection?: 'row' | 'row-reverse' | 'col' | 'col-reverse';
+    flexWrap?: 'wrap' | 'nowrap' | 'wrap-reverse';
+    gridColumns?: number;
```

Comment:
@joneff do we want this to go into the component config ?

Reply:
yes, we do. customers **love** properties